docs/additional-functionality/rapids-udfs.md

```diff
@@ -174,31 +178,80 @@ To enable _GPU Scheduling for Pandas UDF_, you need to configure your spark job
+  |MapInPandasExec|yes|[Map](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#map)|
+  | WindowInPandasExec|yes|[Window](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#series-to-scalar)|
+  | FlatMapGroupsInPandasExec|no|[Grouped Map](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#grouped-map)|
+  | AggregateInPandasExec|no|[Aggregate](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#series-to-scalar)|
+  |FlatMapCoGroupsInPandasExec|no|[Co-grouped Map](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#co-grouped-map)|
+
 
 ### Other Configuration
 
-Following configuration settings are also for _GPU Scheduling for Pandas UDF_
-```
-spark.rapids.python.concurrentPythonWorkers
-spark.rapids.python.memory.gpu.allocFraction
-spark.rapids.python.memory.gpu.maxAllocFraction
-```
+The following configuration settings are also relevant for GPU scheduling for Pandas UDF.
+
+1. Memory efficiency
+
+    ```
+    --conf spark.rapids.python.memory.gpu.pooling.enabled=false \
+    --conf spark.rapids.python.memory.gpu.allocFraction=0.1 \
+    --conf spark.rapids.python.memory.gpu.maxAllocFraction= 0.2 \
+    ```
+    Similar to the [RMM pooling for JVM](../tuning-guide.md#pooled-memory) settings like
+    `spark.rapids.memory.gpu.allocFraction` and `spark.rapids.memory.gpu.maxAllocFraction` except
+    these specify the GPU pool size for the _Python process_. Half of the GPU _available_ memory
+    will be used by default if it is not specified.
+
+
+2. Limit of concurrent Python processes
+
+    ```
+    --conf spark.rapids.python.concurrentPythonWorkers=2 \
+    ```
+    This parameter limits the total concurrent running `Python process` for a Spark executor.
+    It defaults to 0 which means no limit. Note that for certain cases, setting
+    this value too small _may result in  a hang for your Spark job_ because a task may contain
+    multiple PandasUDF(`MapInPandas`) instances which results in multiple Python processes.
+    Each process will try to acquire the Python GPU process semaphore. This may result in a
+    deadlock situation because a Spark job will not proceed until all its tasks are finished
+
+    For example, in a specific Spark Stage that contais 3 PandasUDFs, 2 Spark tasks are running and
+    each task launches 3 Python process while we set this `concurrentPythonWorkers` to 4.
+
+    ```python
+    df_1 = df_0.mapInPandas(udf_1, schema_1)
+    df_2 = df_1.mapInPandas(udf_2, schema_2)
+    df_3 = df_2.mapInPandas(udf_3, schema_3)
+    df_3.explain(True)
+    ```
+    The RAPIDS Accelerator query explain:
+    ```
+    ...
+      *Exec <MapInPandasExec> could partially run on GPU
+        *Exec <MapInPandasExec> could partially run on GPU
+          *Exec <MapInPandasExec> could partially run on GPU
+    ...
+    ```
+
+    ![Python concurrent worker](/docs/img/concurrentPythonWorker.PNG)
```

Comment:
The image does not show up on the jekyll rendered page.  Try changing to: 
```suggestion
    ![Python concurrent worker](../img/concurrentPythonWorker.PNG)
```

Reply:
Done.